### PR TITLE
Various small fixes

### PR DESCRIPTION
**Pull request by leonschoorl**
## Still TODO:

  - [ ] Write a changelog entry (see changelog/README.md)
  - [ ] Check copyright notices are up to date in edited files

[comment]: # (Depending on the change, some of these points may not be necessary. If so, leave the boxes unchecked so it is easier for a reviewer to see what has been omitted.)

### CODE REVIEW

File: clash-ghc/src-ghc/Clash/GHC/LoadModules.hs [hunk 1]
```diff
@@ -501,7 +501,7 @@ loadModules startAction useColor hdl modName dflagsM idirs = do
                   No top-level function called 'topEntity' or 'testBench' found,
                   nor any function annotated with a 'Synthesize' or 'TestBench'
                   annotation. If you want to synthesize a specific binder in
-                  #{show modName}, use '-main-is=myTopEntity'.
+                  #{show modName}, use '-main-is myTopEntity'.
```
Comment (martijnbastiaan) [inline clash-ghc/src-ghc/Clash/GHC/LoadModules.hs:504]: Oof, telling users to do a thing that doesn't work is the worst kind of helpful message :see_no_evil:

File: clash-prelude/src/Clash/Signal/Bundle/Internal.hs [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {-# LANGUAGE CPP #-}
 {-# LANGUAGE TemplateHaskell #-}
 
-module Clash.Signal.Bundle.Internal where
+module Clash.Signal.Bundle.Internal (deriveBundleTuples) where
```
Comment (martijnbastiaan) [inline clash-prelude/src/Clash/Signal/Bundle/Internal.hs:4]: Restricting imports on Internal modules is a bit of an anti-pattern IMO

**Comment by martijnbastiaan** [comment]
Nice batch of changes once more, almost double digits now!

File: clash-ghc/src-ghc/Clash/GHC/LoadModules.hs (hunk 1 above)
Comment (leonschoorl) [reply clash-ghc/src-ghc/Clash/GHC/LoadModules.hs:504]: Or is it the best kind of unhelpful message? :wink:

Comment (martijnbastiaan) [reply clash-ghc/src-ghc/Clash/GHC/LoadModules.hs:504]: Hah

File: clash-prelude/src/Clash/Signal/Bundle/Internal.hs (hunk 2 above)
Comment (leonschoorl) [reply clash-prelude/src/Clash/Signal/Bundle/Internal.hs:4]: Is it?

When I did this I was thinking of it as telling GHC that all the helper functions in this module aren't used else where, so it's free to inline them in `deriveBundleTuples` and doesn't have to keep their core around.

**Comment by leonschoorl** [comment]
I've added some more stuff @alex-mckenna came up with

**Review by martijnbastiaan — APPROVED** [review]
Lookin good :)

**Pull request merged**